TESTS.md

```diff
@@ -852,6 +852,18 @@ data Link c1 c2 a c =
             Link (Proxy b)
 ```
 
+ttuegel qualified infix sections get mangled #273
+
+```haskell
+-- https://github.com/chrisdone/hindent/issues/273
+import qualified Data.Vector as V
+
+main :: IO ()
+main = do
+  let _ = foldr1 V.++ [V.empty, V.empty]
```

Comment:
I have fixed it. It was caused by the wrong pretty printing for `Var` in `exp` function. I also have added there more corner cases to regression test.